arm/Microsoft.Authorization/policyAssignments/.parameters/mg.parameters.json

```diff
@@ -41,7 +41,7 @@
         },
         "notScopes": {
             "value": [
-                "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg"
+                "/subscriptions/<<subscriptionId>>/resourceGroups/<<resourceGroupName>>"
```

Comment:
Do we have this token? Haven't seen it used anywhere else

Reply:
A couple of resources in the MS AuthZ namespace use it. We already support it and it's in our documentation. It replaces it with the resource group name from the YAML global variables. I can replace it back to the hard coded version if wanna keep using that?

Reply:
I'd also suggest to have consistency on that and decide whether to use it everywhere or not.

Reply:
I'll revert it and we can decide to roll it out for all pipelines in another issue instead as it isn't the reason for this PR.